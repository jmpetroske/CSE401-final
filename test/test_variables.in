$var = 12px;
$var2 = 12px * 2;

.selector {
    height: $var;
    width: $var2;
    $var = 0;
}

.selector2 {
    height: $var;
    width: $var2;
}
