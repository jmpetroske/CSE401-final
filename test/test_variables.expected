.selector {
height: 12px;
width: 24px;
}
.selector2 {
height: 12px;
width: 24px;
}
